user-preferences-explainer.md

```diff
@@ -0,0 +1,68 @@
+# User Preferences Explainer
+
+Authors: [Aaron Gustafson](https://github.com/aarongustafson), Louise Brett (loubrett@google.com)
+
+## Overview
+
+Currently in the web app manifest a `theme_color` and `background_color` can be defined. However there is no way to change these based on dark mode or [other user preferences](#possible-extensions).
+
+This document proposes a new `user_preferences` manifest member that enables web apps to provide alternate values for manifest members given specific user preferences. The structure of this `user_preferences` field matches the structure of the proposed <code>[translations](https://github.com/WICG/manifest-incubations/blob/gh-pages/translations-explainer.md)</code> member.
+
+This new member is inspired by the [CSS media queries user preferences](https://drafts.csswg.org/mediaqueries-5/#mf-user-preferences). The keys are simple strings but they are derived from the CSS media query syntax and can easily be converted back if necessary.
+
+## Proposal
+
+Add a new dictionary manifest entry `user_preferences`, mapping preference strings to a [`ManifestOverride` object](#manifestoverride-object).
+
+Initially, the valid keys of the `user_preferences` member are `color_scheme_dark` and `color_scheme_light`. This could be expanded in the future to include other [user preference media features](#possible-extensions).
+
+### `ManifestOverride` Object
+
+The `ManifestOverride` is a generic object that contains a subset of redefined manifest properties appropriate to the context (e.g. `user_preferences`) in which the `ManifestOverride` is being used. The properties which may be redefined in the `ManifestOverride` object depend on the context (e.g. `user_preferences`). Any properties not allowed within the context will be ignored. The redefined fields override the values set in the root of the manifest.
+
+For the `user_preferences` member, the acceptable keys for the `ManifestOverride` include:
+
+*   `theme_color`
+*   `background_color`
+*   `icons`
+    *   `src`
+    *   `type`
+*   `shortcuts`
+    *   `icons`
+        *   `src`
+        *   `type`
+
+### Example
+
+```json
+"user_preferences": {
+    "color_scheme_dark": {
+        "theme_color": "#000"
+        "background_color": "#000"
+    },
+    "color_scheme_light": {
+        "theme_color": "#fff"
+        "background_color": "#fff"
+    }
+  }
+```
+
+When a user has dark mode enabled, the fields redefined for `color_scheme_dark` will be used instead of the top level fields.
+
```

Comment:
You may also want an "alternatives considered" section to explain why you aren't using generic css queries as user preference selectors?